src/components/safe-messages/Msg/styles.module.css

```diff
@@ -2,3 +2,10 @@
   font-weight: 700;
   text-decoration: underline;
 }
+
+.truncated {
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```

Comment:
`max-height: 10ex` + `text-overflow: ellipsis` would probably work also in Firefox.

Reply:
```css
max-height: 9ex;
display: block;
overflow: hidden;
```

works but `text-overflow: ellipsis` does not work in this variant.

According to https://caniuse.com/?search=line-clamp the `-webkit-line clamp` property seems to be supported by all important browsers.

I added `max-height: 9ex;` to the current css in case some browsers do not support line-clamp.